samples/12.customization-minimizable-web-chat/README.md

```diff
@@ -0,0 +1,579 @@
+# Sample - Customize Web Chat by Making the Component Minimizable
+
+## Description
+
+A web page where instead of Web Chat being the primary app, the Web Chat component is nested in a minimizable area of the DOM. This is a popular feature for Web Chat users, where the bot is an accessory to the website, not the main feature.
+
+This app is built with `create-react-app`. If you have not used this package before, check out the [further reading](#further-reading) section before you get started.
+
+# Test out the hosted sample
+
+- [Try out MockBot](https://microsoft.github.io/BotFramework-WebChat/12.customization-minimizable-web-chat)
+
+# How to run locally
+
+- Fork this repository
+- Navigate to `/Your-Local-WebChat/samples/12.customization-minimizable-web-chat` in command line
+- Run `npm install`
+- Run `npm start`
+- Browse to [http://localhost:3000/](http://localhost:3000/)
+
+# Things to try out
+
+- Type `card bingsports` and notice the Adaptive Card
+- Minimize the Web Chat component by clicking the minimize button Web Chat header
+- Click the Web Chat button to toggle Web Chat (visually, this is the blue button on the bottom right side of the page)
+- Note that conversation state is preserved
+
+# Code
+
+> Jump to [completed code](#completed-code) to see the end-result `App.js`, `MinimizableWebChat.js`, and `WebChat.js`.
+
+
+## Overview
+
+This sample is the first of the 'advanced' tutorials, where Web Chat evolves beyond a single `index.html`. We will be using the node package manager, and installing several dependencies into an app that is made up of multiple files.
+
+> Please note that while this sample does have `.css` files, they will not be discussed in this overview, though the `jsx` code snippets will include class names for reference purposes.
+
+First let's set up the project.
+
+```sh
```

Comment:
You might want to consider adding a step here that navigates to the right directory before running these commands.

Reply:
ex:

```
cd some/initial/directory
npx create-react-app ...
...
```